Ginger/Ginger/Variables/VariableEditPage.xaml.cs

```diff
@@ -77,17 +77,25 @@ public VariableEditPage(VariableBase v, Context context, bool setGeneralConfigsA
             GingerCore.GeneralLib.BindingHandler.ObjFieldBinding(cbSetAsInputValue, CheckBox.IsCheckedProperty, mVariable, nameof(VariableBase.SetAsInputValue));
             GingerCore.GeneralLib.BindingHandler.ObjFieldBinding(cbSetAsOutputValue, CheckBox.IsCheckedProperty, mVariable, nameof(VariableBase.SetAsOutputValue));
 
-            if (mode ==eEditMode.Global)
+            if (mode ==eEditMode.Global || mode==eEditMode.SharedRepository)
```

Comment:
With this condition set as input and set as outpit check boxes will be hidden when opening variable from Shared repository.
But this check boxes should be hidden only for edit mode global and not for shared repository